biologic_mpr.yml

```diff
@@ -0,0 +1,17 @@
+id: >-
+  biologic-mpr
+name: >-
+  Biologic MPR file
+description: >-
+  A binary data file created by Biologic cyclers, containing time- and frequency-domain data pertaining to many electrochemical techniques, such as voltammetry, potentiometry, coulometry and impedance spectroscopy.
```

Comment:
Question about "code" style: Should we be running those files through `black` or some other formatter on commit? Or are lines of arbitrary length, such as those descriptions, OK?

Reply:
I was thinking of adding `yamlfmt` via pre-commit CI at a later date, but can just do it now